tsconfig.json

```diff
@@ -30,7 +30,10 @@
     "forceConsistentCasingInFileNames": true,  /* Disallow inconsistently-cased references to the same file. */
     "paths": {
       "*": [
-        "../node_modules/*"
+        "../node_modules/*",
+      ],
```

Comment:
This should be the right mode to resolving alias with typescript and webpack, following this article https://stackoverflow.com/questions/40443806/webpack-resolve-alias-does-not-work-with-typescript but according to one of the answers is possible to use `@babel/preset-typescript` without `ts-loader`